pycares/_cfficore/__init__.py

```diff
@@ -0,0 +1,667 @@
+#!/usr/bin/env python
+
+from _pycares_cffi import ffi as _ffi, lib as _lib
```

Comment:
pycares/_cfficore/pycares_build.py will automatic generate this _pycares_cffi.so module by call cff.FFI.set_source()